src/main/resources/application.properties

```diff
@@ -1,5 +1,7 @@
-#define o diretório de criação do certificado em pdF
+#define o diretório de criação do certificado em PDF
 certificate.output.directory=out/test/test
+#define o diretório de armazenamento das imagens do PDF
+certificate.images.directory=src/main/resources/images
```

Comment:
Eu acredito que isso não vai funcionar quando estivermos executando fora da IDE. 

Procure como carregar arquivos vindos do classpath corretamente
https://stackoverflow.com/questions/44399422/read-file-from-resources-folder-in-spring-boot